### PR TITLE
113538: defining the contract for the decision outcome apis

### PR DESCRIPTION
**Pull request by mikestock-nimble**
**What is the change?**

Defining the contract for the decision outcome apis

Created skeleton apis and api tests

**Why do we need the change?**

**What is the impact?**

**Azure DevOps Ticket**

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/CreateDecisionOutcomeRequest.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.API.Contracts.Decisions.Outcomes
+{
+	public record CreateDecisionOutcomeRequest : DecisionOutcome
+	{
```
Comment (emma-w-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/CreateDecisionOutcomeRequest.cs:10]: I guess this is to be done later?

File: ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/UpdateDecisionOutcomeRequest.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ConcernsCaseWork.API.Contracts.Decisions.Outcomes
+{
+	public record UpdateDecisionOutcomeRequest : DecisionOutcome
+	{
```
Comment (emma-w-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/UpdateDecisionOutcomeRequest.cs:10]: Again, I'm guessing this is to be done later?

File: ConcernsCaseWork/ConcernsCaseWork.API.Tests/Controllers/DecisionOutcomeControllerTests.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+using System.Net;
+using System.Net.Http;
+using System.Net.Mime;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace ConcernsCaseWork.API.Tests.Controllers
+{
+	public class DecisionOutcomeControllerTests : IDisposable
```
Comment (emma-w-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.API.Tests/Controllers/DecisionOutcomeControllerTests.cs:23]: Is this in the right place? The other controller tests in this namespace use mocks. Should this be in the '..tests.integration' namespace?

File: ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/CreateDecisionOutcomeRequest.cs (hunk 1 above)
Comment (mikestock-nimble) [reply ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/CreateDecisionOutcomeRequest.cs:10]: Agreed offline

File: ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/UpdateDecisionOutcomeRequest.cs (hunk 2 above)
Comment (mikestock-nimble) [reply ConcernsCaseWork/ConcernsCaseWork.API.Contracts/Decisions/Outcomes/UpdateDecisionOutcomeRequest.cs:10]: Agreed offline

File: ConcernsCaseWork/ConcernsCaseWork.API.Tests/Controllers/DecisionOutcomeControllerTests.cs (hunk 3 above)
Comment (mikestock-nimble) [reply ConcernsCaseWork/ConcernsCaseWork.API.Tests/Controllers/DecisionOutcomeControllerTests.cs:23]: Will look into this as we go forward to see if we can get them stable

**Pull request merged**